handsontable/src/helpers/dom/element.js

```diff
@@ -1194,3 +1194,15 @@ export function runWithSelectedContendEditableElement(element, callback, invisib
 
   removeContentEditableFromElementAndDeselect(element, invisibleSelection);
 }
+
+/**
+ * Check if the element is HTMLElement.
+ *
+ * @param {HTMLElement|number} element Element to check.
+ * @returns {boolean} `true` if the element is HTMLElement.
+ */
+export function isHTMLElement(element) {
```

Comment:
Can you add tests for that helper? The logic can be put here https://github.com/handsontable/handsontable/blob/feature/dev-issue-2243/handsontable/src/helpers/dom/__tests__/element.unit.js. Also worth to add a [typing](https://github.com/handsontable/handsontable/blob/feature/dev-issue-2243/handsontable/types/helpers/dom/element.d.ts) and test it as well (https://github.com/handsontable/handsontable/blob/feature/dev-issue-2243/handsontable/src/helpers/dom/__tests__/dom.types.ts).